### DISCUSSION TITLE
绳匠们，我这台电老能运行Fairy吗？

### DISCUSSION THREAD

**Discussion by hengguocunzhuang** [share121/inter-knot#2615] [其他]
![任务管理器 2024_8_18 0_43_50](https://github.com/user-attachments/assets/449495ca-1c66-45ed-854e-15e03ed591ad)

**Comment by ghost-feng** [comment]
不行，太烧显卡了

**Comment by JobNumber-9527** [comment]
fairy的耗电量估计能直接给你电闸跳了<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by ysd1911** [comment]
前提是你要交得起电费OAO
